nubis/puppet/snippets_stats.pp

```diff
@@ -0,0 +1,98 @@
+cron::daily { "${project_name}-snippets_stats":
+  user    => 'etl',
+  command => "nubis-cron ${project_name}-snippets_stats /opt/etl/snippets_stats/run",
+}
+
+python::pyvenv { "${virtualenv_path}/snippets_stats" :
+  ensure  => present,
+  version => '3.4',
+  require => [
+    File[$virtualenv_path],
+  ],
+}
+
+# Install Mozilla's snippets-stats
+python::pip { 'snippets-stats':
+  ensure     => 'present',
+  virtualenv => "${virtualenv_path}/snippets-stats",
+  url        => 'git+https://github.com/mozilla-it/snippets-stats@78c299062701afcb9df60feb5b91bb5faba74cb8',
```

Comment:
This repo isn't pip installable, it's missing a setup.py at a minimum to drive installation.

```
$> pip install --user git+https://github.com/mozilla-it/snippets-stats@78c299062701afcb9df60feb5b91bb5faba74cb8
Collecting git+https://github.com/mozilla-it/snippets-stats@78c299062701afcb9df60feb5b91bb5faba74cb8
  Cloning https://github.com/mozilla-it/snippets-stats (to 78c299062701afcb9df60feb5b91bb5faba74cb8) to /tmp/pip-OopbEX-build
  Could not find a tag or branch '78c299062701afcb9df60feb5b91bb5faba74cb8', assuming commit.
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    IOError: [Errno 2] No such file or directory: '/tmp/pip-OopbEX-build/setup.py'